template/.github/ISSUE_TEMPLATE/publish_component.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Publish Component
+about: Create a ticket to drive compliance before publishing a a new componet to Auro Design System
+title: 'auro-[name]; publish to Auro Design System'
+labels: 'Status: Proposal, not-reviewed'
+---
+
+# Publish Component
+
+## Specific steps required to publish this component.
+
+All of the below outlined steps must be completed.
+
+### Design & Planning
+- [ ] Reviewed [AuroLabs documentation](http://auro.alaskaair.com/)
+- [ ] Reviewed [Auro's Contributing Guidelines](http://auro.alaskaair.com/contributing)
+- [ ] Reviewed [Auro's CSS conventions](http://auro.alaskaair.com/webcorestylesheets/conventions) and how to apply CSS to custom elements
+- [ ] Design team engaged in designing the new component
+- [ ] Design work integrated in the Auro Design System UI Kit
+
+### Development
+- [ ] Component built using latest version of [WC-Generator](http://auro.alaskaair.com/getting-started/developers/generator/install)
+- [ ] Addresses all design and UX as proposed in the design comp
+- [ ] A11y in regards to role and live notifications need to be coded into the final WC
+- [ ] All style code follows [Auro's Style Guide](http://auro.alaskaair.com/webcorestylesheets/guidelines)
+
+### Documentation
+- [ ] Demo page created for the Auro Doc Site
+- [ ] Examples and use case info
+- [ ] Bundled version info
+- [ ] README doc completed with all relative information
+- [ ] API documentation created
+
+### Testing
+- [Browser compatibility](http://auro.alaskaair.com/support/browsersSupport) - desktop and mobile versions
+  - [ ] Chrome
+  - [ ] Firefox
+  - [ ] Edge (79 +)
+  - [ ] Safari
+- [Accessibility](http://auro.alaskaair.com/a11y-statement)
```

Comment:
Should be updated to say "one or more of the following"

Reply:
The browser list? Considering differences in Safari vs Everyone else and mobile vs. non-mobile experience do you think that testing against a single browser is suffecient?

Reply:
No, I was referring to the accessibility options.